OtripleS.Web.Api.Tests.Unit/Services/GuardianAttachments/GuardianAttachmentServiceTests.Exceptions.Add.cs

```diff
@@ -0,0 +1,123 @@
+﻿//---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+//----------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using OtripleS.Web.Api.Models.GuardianAttachments;
+using OtripleS.Web.Api.Models.GuardianAttachments.Exceptions;
+using Xunit;
+
+namespace OtripleS.Web.Api.Tests.Unit.Services.GuardianAttachments
+{
+    public partial class GuardianAttachmentServiceTests
+    {
+        [Fact]
+        public async Task ShouldThrowDependencyExceptionOnAddWhenSqlExceptionOccursAndLogItAsync()
+        {
+            // given
+            GuardianAttachment randomGuardianAttachment = CreateRandomGuardianAttachment();
+            GuardianAttachment inputGuardianAttachment = randomGuardianAttachment;
```

Comment:
Just define one variable and call it `someGuardianAttachment` - since in the case of exceptions it's just a filler value not the main focus of the test.

Reply:
Do the same for the rest please.

Reply:
Resolved